Signal/src/AppDelegate.m

```diff
@@ -359,6 +406,15 @@ - (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull N
             return NO;
         }
 
+        if ([Environment getCurrent].phoneManager.hasOngoingCall) {
+            DDLogWarn(@"%@ ignoring INStartAudioCallIntent due to ongoing RedPhone call.", self.tag);
+            return NO;
+        }
+        if ([Environment getCurrent].callService.call != nil) {
+            DDLogWarn(@"%@ ignoring INStartAudioCallIntent due to ongoing WebRTC call.", self.tag);
+            return NO;
+        }
+
```

Comment:
`OutboundCallInitiator` is asynchronous - i.e. it does the "supports webrtc?" request before actually starting a call. We might want to modify it to make this check as well before starting a call. Hmm, I guess I'll do that.